queries/gleam/highlights.scm

```diff
@@ -0,0 +1,158 @@
+; Keywords
+[
+  "as"
+  "const"
+  "external"
+  "let"
+  "opaque"
+  "pub"
+  "todo"
+  "try"
```

Comment:
Is try also more in direction of `@exception` or more like "launch" or general control flow?

Reply:
It's flow control, similar to this Go code:

```go
if err != nil {
        return err
}
```